styles/figures.css

```
@@ -1,11 +1,14 @@
-figure.quote figcaption {
+figure.fig-quote figcaption {
   text-align: right;
 }
-figure.quote figcaption > p:before {
+figure.fig-quote figcaption > p:before {
   content: '—';
   padding-right: 0.5em;
 }
-figure.code > div {
+figure.fig-quote.pull-quote > blockquote { 
+  font-size: 1.25em;
```

In addition to being a bit bigger, I think that these should also be in rems!

```suggestion
  font-size: 1.25rem;
```